test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -626,6 +636,92 @@ public void notSettingStatusDoesntClearStatus() {
         verify(account, never()).setStatus(any());
     }
 
+    @Test
+    public void updateParticipantEmailCannotBeChanged() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval();
+        when(account.getEmail()).thenReturn("persisted@email.com");
+        when(account.getEmailVerified()).thenReturn(Boolean.TRUE);
+        
+        StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT)
+                .withEmail("updated@email.com").withEmailVerified(Boolean.FALSE).build();
+        
+        participantService.updateParticipant(STUDY, ImmutableSet.of(), participant);
+        
+        verify(accountDao).updateAccount(accountCaptor.capture());
+        Account account = accountCaptor.getValue();
+        verify(account, never()).setEmail("updated@email.com");
+        verify(account, never()).setEmailVerified(Boolean.FALSE);
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(STUDY, ID, "updated@email.com");
```

Comment:
It might be worthwhile to check that we never call setEmail(anyString()) and sendEmailVerificationToken(anyString()). This will test that we don't also send an email verification email to the original email address either.

Reply:
Makes sense, changed.